### PR TITLE
Running E2E Tests External Contributors (Mostly Docs)

### LINKED ISSUE #2271: Improve CI experience for external contributors
### Description
Currently it is difficult for external contributors to contribute to the wallet repo and feel confident that they are not introducing breaking changes because they cannot run the e2e tests due to not having secrets. We should re-evaluate if the secrets are necessary and remove or create a workaround.

### Acceptance criteria
- [ ] External contributors should be able to run the e2e tests from a forked version of the repo

Relates to #2186

### PR DESCRIPTION
**Pull request by MuckT**
### Description

Updates the `packges/mobile/README.md` and e2e tests to make running the wallet repository easier for outside contributors.

### Other changes

N/A

### Tested

This has been tested by downloading a zip of the wallet repo and running without branding access.

### How others should test

This does not need to be tested by QA. Reviewers should download the wallet repository via zip and see that they can run the app and e2e tests without branding or secrets access.

### Related issues

- Fixes #2271

### Backwards compatibility

Yes

### CODE REVIEW

File: packages/mobile/README.md [hunk 1]
```diff
@@ -369,8 +369,9 @@ See [`src/identity/verification.test.ts`] for an example.
 We use [Detox][detox] for E2E testing. In order to run the tests locally, you
 must have the proper emulator set up. Follow the instructions in [e2e/README.md][e2e readme].
 
-Once setup is done, you can run the tests with `yarn test:e2e:android` or `yarn test:e2e:ios`.
-If you want to run a single e2e test: `yarn test:e2e:ios -f Exchange.spec.js -t "Then Buy CELO"`
+Once setup is done, you can buil the tests with `yarn e2e:build:android-release` or `yarn e2e:build:ios-release`.
```
Comment (kathaypacific) [inline packages/mobile/README.md:372]: `build` :D

File: packages/mobile/README.md [hunk 2]
```diff
@@ -94,7 +94,7 @@ _If you are a Valora employee, please ask to be added to the Valora iOS developm
 
 Xcode is needed to build and deploy the mobile wallet to your iOS device. If you do not have an iOS device, Xcode can be used to emulate one.
 
-Install [Xcode 12.2](https://developer.apple.com/download/more/?q=xcode) (an Apple Developer Account is needed to access this link).
+Install [Xcode 13.2](https://developer.apple.com/download/more/?q=xcode) (an Apple Developer Account is needed to access this link).
```
Comment (kathaypacific) [inline packages/mobile/README.md:97]: Maybe we can just have `Xcode 13` here? I am using 13.3 now and it seems to work okay, so not sure that there's much reason to also specify the minor version here?

**Review by kathaypacific — APPROVED** [review]
Thank you! This is great 🥳 just a couple minor comments from me.

Was also wondering if the title can be updated to be more specific, and indicate that this change is mainly a doc update? Thinking that it would be easier to understand what to expect from this change when reading the change log or commit history later :)

File: packages/mobile/README.md (hunk 2 above)
Comment (MuckT) [reply packages/mobile/README.md]: Fixed in https://github.com/valora-inc/wallet/pull/2381/commits/92a6d9afafabfbd72bb588ff4cd66ffec4fce54d!

File: packages/mobile/README.md (hunk 1 above)
Comment (MuckT) [reply packages/mobile/README.md]: Fixed in https://github.com/valora-inc/wallet/pull/2381/commits/92a6d9afafabfbd72bb588ff4cd66ffec4fce54d!

**Comment by jeanregisser** [comment]
@mergifyio refresh

**Comment by ValoraQA** [comment]
Hey @MuckT since it is not to be tested by QA we are skipping the PR
Thanks..

**Pull request merged**